doc/contributing/documentation_guide.md

```diff
@@ -217,57 +217,83 @@ For methods written in C, \RDoc cannot determine what arguments
 the method accepts, so those need to be documented using \RDoc directive
 [`:call-seq:`](rdoc-ref:RDoc::Markup@Method+arguments).
 
+For a singleton method, use the form:
+
+```c
+class_name.method_name(method_args) {|block_args| ... } -> return_type
+```
+
 Example:
 
 ```c
-*  call-seq:
-*    array.count -> integer
-*    array.count(obj) -> integer
-*    array.count {|element| ... } -> integer
+*  :call-seq:
+*    Hash.new(default_value = nil) → new_hash
+*    Hash.new {|hash, key| ... } → new_hash
```

Comment:
We should use `->` rather than the right allow symbol `→`.

Reply:
Fixed.